wolfcrypt/src/wc_port.c

```diff
@@ -126,6 +127,161 @@ int wolfCrypt_Cleanup(void)
     return ret;
 }
 
+#if !defined(NO_FILESYSTEM)
+
+int wc_ChangeDir(const char* path)
+{
+    int ret;
+#ifdef USE_WINDOWS_API
+    ret = SetCurrentDirectoryA(path);
+#else
+    ret = chdir(path);
```

Comment:
I'm going to revert this part of the change. It was only used in test.h and there isn't a good reason to add this API.